src/sections/Related-Posts/relatedPosts.style.js

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+
+
+const RelatedPostsWrapper = styled.div`
+        .sidebar-widgets {
+            padding: 0 30px;
+        }
+
+        .widgets-title{
+            margin-bottom: 20px;
+            margin-left: 5px;
+            h3{
+                display: inline-block;
+                font-size: 21px;
+                font-weight: 500;
+                line-height: 30px;
+                position: relative;
+                &:before{
+                    position: absolute;
+                    top: 50%;
+                    right: -80px;
+                    content: '';
+                    width: 70px;
+                    height: 1px; 
+                }
+            }
+            
+        }
+        
+        .sidebar-widgets{
+            &+.sidebar-widgets{
+                margin-top: 52px;
+            }
+            ul{
+                margin: 0;
+                padding: 0;
+                list-style: none;
+            }
+            
+            &.tags{
```

Comment:
Even these can be removed right

Reply:
We have list of related posts there, so these tags are effective, aren't they?

Reply:
I don't think so. You can try to remove and check if makes any difference

Reply:
Can you tell me exactly which classes to check for :)